checkov/kubernetes/parser/k8_json.py

```diff
@@ -53,11 +49,8 @@ def load(filename: Path) -> Tuple[List[Dict[str, Any]], List[Tuple[int, str]]]:
     return template, file_lines
 
 
-class SafeLineLoader(SafeLoader):
-    def construct_mapping(self, node: MappingNode, deep: bool = False) -> dict[Hashable, Any]:
-        mapping = super().construct_mapping(node, deep=deep)
-        # Add 1 so line numbering starts at 1
-        # mapping['__line__'] = node.start_mark.line + 1
-        mapping['__startline__'] = node.start_mark.line + 1
-        mapping['__endline__'] = node.end_mark.line + 1
-        return mapping
+class CustomDecoder(SimpleDecoder):
+    def object_hook(self, obj: dict[str, Any]) -> Any:
+        obj[START_LINE] = 0
```

Comment:
why are you specifically marking it with 0? isn't it important to actually use the correct line number?

Reply:
it's a fallback for large files, similar to what we do in cfn jsons